admin/modules/tools/file_verification.php

```diff
@@ -71,6 +71,16 @@
 				}
 			}
 
+			if(!file_exists(MYBB_ROOT."inc/plugins/hello.php") && $parts[1] == "./inc/plugins/hello.php")
+			{
+				continue;
+			}
+
+			if(!is_dir(MYBB_ROOT."install/") && substr($parts[1], 0, 9) == "./install")
```

Comment:
Can you extend the second condition by a trailing slash to make sure it's the directory?